Farm&Gun/Assets/Scripts/Entities/Player/PlayerMovement.cs

```diff
@@ -36,7 +36,7 @@ void Update()
         if (animator.GetBool("Dead")) return;
 
         if (!animator.GetBool("Performing Attack")) EnableMovement();
```

Comment:
Could you move EnableMovement(); to the next line? I think it would be a bit more readable.